File: 3582-find-indices-of-stable-mountains/find-indices-of-stable-mountains.js
/**
 * @param {number[]} height
 * @param {number} threshold
 * @return {number[]}
 */
var stableMountains = function(h, t) {
    let ans = [];

    for(let i=h.length-1; i>0; i--){
        let curr = h[i];
        let prev = h[i-1];

        if(prev>t){
            ans.push(i)
        }
    }

    return ans;
};

File: 3582-find-indices-of-stable-mountains/find-indices-of-stable-mountains.go
func stableMountains(h []int, t int) []int {
    var arr []int;

    for i:= len(h)-1; i>0; i-- {
        prev := h[i-1]

        if(prev>t){
            arr = append(arr,i)
        }
    }

    return arr;
}